addons/event/models/event_registration.py

```diff
@@ -64,46 +80,26 @@ def _check_ticket_seats_limit(self):
             if record.event_ticket_id.seats_max and record.event_ticket_id.seats_available < 0:
                 raise ValidationError(_('No more available seats for this ticket'))
 
-    @api.onchange('partner_id')
-    def _onchange_partner(self):
-        if self.partner_id:
-            self.update(self._synchronize_partner_values(self.partner_id))
+    @api.constrains('event_id', 'event_ticket_id')
+    def _check_event_ticket(self):
```

Comment:
How can this happen exactly?
A note would be helpful.

Reply:
Ah voilà. There was an onchange resetting the ticket when changing the event. I changed it by a constraint as having a computed field just to reset a value in a corner case would be too heavy. Indeed normally you don't change the event of a registration, but if it occurs and tickets do not match it gueule.